com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/java/com/ca/lsp/core/cobol/preprocessor/sub/document/CopybookResolution.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+
+package com.ca.lsp.core.cobol.preprocessor.sub.document;
+
+import com.ca.lsp.core.cobol.model.CopybookModel;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/** This interface represents the API for retrieving copybooks */
+public interface CopybookResolution {
```

Comment:
Adjectives is used for Interface naming  -  ResolvableCopybook

Reply:
I disagree - this object is not a copybook itself, is just a helper for them

Reply:
@temanbrcom It is Java naming rule, existing even from Sun time.

Reply:
3. Interface naming convention
In Java, interfaces names, generally, should be adjectives. Interfaces should be in titlecase with the first letter of each separate word capitalized. In same cases, interfaces can be nouns as well when they present a family of classes e.g. List and Map.
[source](https://howtodoinjava.com/java/basics/java-naming-conventions/#:~:text=Interface%20naming%20convention,classes%20e.g.%20List%20and%20Map%20.)

Here, I recognize this interface as a representation of a family of classes